backend/typescript/prisma/schema.prisma

```diff
@@ -12,25 +12,25 @@ model test {
 }
 
 model users {
-  id             String          @id @default(auto()) @map("_id") @db.ObjectId
-  v              Int             @map("__v")
+  id             String           @id @default(auto()) @map("_id") @db.ObjectId
+  v              Int              @map("__v")
   authId         String
   email          String
   firstName      String
   lastName       String
   role           String
+  kitchenRequest kitchenRequest[]
 }
 
-//tina-emma/kitchen-request-model
 model kitchenRequest {
   id            String   @id @default(auto()) @map("_id") @db.ObjectId
   v             Int      @map("__v")
   requestName   String
-  requester     User     @relation(fields: [requesterId], references: [id])
-  requesterId   String     
+  requester     users    @relation(fields: [requesterId], references: [id])
```

Comment:
Not sure if we'd like to keep this `requester` relation & `kitchenRequest` in users model above, or have only `requesterId` as the link to the users model for simplicity.

Reply:
Looks good!